samplers/parser.go

```diff
@@ -72,9 +72,13 @@ func ParseMetricSSF(metric *ssf.SSFSample) (*UDPMetric, error) {
 		return nil, invalidMetricTypeError
 	}
 	h.Write([]byte(ret.Type))
-	ret.Value = float64(metric.Value)
+	if metric.Metric == ssf.SSFSample_SET {
+		ret.Value = metric.Message
+	} else {
+		ret.Value = float64(metric.Value)
+	}
 	ret.SampleRate = metric.SampleRate
-	tempTags := make([]string, len(metric.Tags))
+	tempTags := make([]string, 0)
```

Comment:
I mean that it fixed a big. I was poking around recently and managed to get a metric with a nil `Tags` and subsequent code assumed it wasn't nil. I might be tracing poorly, but I think this was my idea for a fix!